app/_src/networking/meshservice.md

```diff
@@ -64,19 +69,65 @@ MeshService. For this reason, Kuma generates `MeshServices` from `Services` and:
 - reuses VIPs in the form of cluster IPs
 - uses Kubernetes DNS names
 
+{% if_version lte:2.8.x %}
 {% tip %}
 You need to set the `kuma.io/mesh` label on any `Services` from which
 a `MeshService` should be generated.
 {% endtip %}
+{% endif_version %}
 
 In the vast majority of cases, Kubernetes users do not create `MeshServices`.
 
 ### Universal
 
+{% if_version lte:2.8.x %}
 In universal zones, `MeshServices` need to be created manually for now. A
 strategy of
 automatically generating `MeshService` objects from `Dataplanes` is planned for
 the future.
+{% endif_version %}
+
+{% if_version gte:2.9.x %}
+In universal zones, `MeshServices` are generated based on the `kuma.io/service`
+value of the `Dataplane` `inbounds`. The only restriction in this case is that
+the port numbers match. For example an inbound:
+
+```
+      inbound:
+      - name: main
```

Comment:
I think we should add a note what is the default value of the field `name` if not provided?

Reply:
done